infra/build/functions/project_sync.py

```diff
@@ -35,7 +35,7 @@
 DEFAULT_BUILDS_PER_DAY = 1
 MAX_BUILDS_PER_DAY = 4
 COVERAGE_SCHEDULE = '0 6 * * *'
-INTROSPECTOR_SCHEDULE = '0 0 * * *'
+INTROSPECTOR_SCHEDULE = '0 8 * * *'
```

Comment:
Should we make this like 10 instead? To give a bit more time for coverage reports to complete.

Reply:
Yes, as I checked coverage build finish within 1 hour. WDYT? I can set it to 10.

Reply:
Let's give it a bit more leeway here and be safe.

Reply:
sure.